soc/nxp/kinetis/ke1xz/Kconfig

```diff
@@ -0,0 +1,53 @@
+# Kinetis KE1xZ series MCU
+
+# Copyright (c) 2019 Vestas Wind Systems A/S
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_KINETIS_KE1XZ
+	select ARM
+	select CPU_CORTEX_M0PLUS
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select CPU_CORTEX_M_HAS_VTOR
+	select CLOCK_CONTROL
+	select HAS_MCUX
+	select HAS_MCUX_FTFX
+	select HAS_MCUX_LPI2C
+	select HAS_MCUX_LPSPI
+	select HAS_MCUX_LPUART
+	select HAS_MCUX_PCC
+	select HAS_MCUX_RTC
+	select HAS_MCUX_SIM
+	select HAS_MCUX_ADC12
+	select HAS_MCUX_SCG
+	select HAS_MCUX_WDOG32
+	select HAS_MCUX_FTM
+	select HAS_MCUX_LPTMR
+	select HAS_MCUX_EDMA
+	select HAS_MCUX_ACMP
+	select HAS_MCUX_PWT
+	select HAS_MCUX_RCM
+	select PLATFORM_SPECIFIC_INIT
+	select HAS_PM
+
+if SOC_SERIES_KINETIS_KE1XZ
+
+config WDOG_ENABLE_AT_BOOT
+	bool "Keep watchdog timer enabled at boot"
+	help
+	  Keep the watchdog timer enabled at boot with the internal
+	  128kHz LPO clock (and a prescaler of 256) as clock
+	  source. The application can take over control of the
+	  watchdog timer after boot and install a different timeout,
+	  if needed.
+
+config WDOG_INITIAL_TIMEOUT
+	int "Initial timeout for the watchdog timer in milliseconds"
+	depends on WDOG_ENABLE_AT_BOOT && WDOG_INIT
+	range 2 131070
+	default 2048
+	help
+	  Initial timeout value for the watchdog timer in
+	  milliseconds.
```

Comment:
Not SoC settings, move to `drivers/watchdog/*`

Reply:
Unresolved

Reply:
@nordicjm this kconfig is already existing and tied to z_arm_watchdog_init in reset.S for the arch, I don't think it belongs with a driver because drivers cannot run in early reset. This is a platform level kconfig. I think there is some technical debt here with the kconfig architecture being bad though

Reply:
I would like if we can fix this bad kconfig architecture but I feel it is outside the scope of this PR

Reply:
@EmilioCBen  I think actually you should just remove this kconfig from this PR, because it is clearly coming from a copy paste of the ke1xf, and that config for that SOC appears not to be used in tree, must be added for a downstream user (@henrikbrixandersen probably). I think for this SOC we don't strictly need to add these configs